src/demos/calcite-pick-list.html

```diff
@@ -10,21 +10,21 @@
     <calcite-demo-nav page-id="calcite-pick-list"></calcite-demo-nav>
     <section class="example-container">
       <calcite-pick-list id="one" text-heading="Selection Pick List - Multi" multiple>
-        <calcite-pick-list-item text-heading="T. Rex" text-description="arm strength impaired" value="trex">
+        <calcite-pick-list-item text-heading="T. Rex" text-description="arm strength impaired" value="trex"  metadata='{"category":"dinosaur", "tags":["carnivore"]}'>
```

Comment:
I think we should update the samples to set the metaData on the node rather than through the attribute since that isn't valid because its an object. We don't want authors doing this.

Reply:
Agreed. I thought we had decided to not support complex data through attributes.

Reply:
I've updated the code to pass the metadata via a property.

These sorts of implementation details will really be in the hands of the mapmaker team. The web component will support accepting rich data via any valid mechanism, and it will get parsed.

So the rich data is supported as a prop of the component. It can be passed in as either an attribute (which will probably make the most sense for those using JSX) or a property set via dot notation.

Reply:
We don't want the mapmaker team thinking they can do this without JSX which is what it was doing above.